packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -214,6 +214,20 @@ abstract class WidgetsBindingObserver {
   ///    boilerplate.
   void didChangeTextScaleFactor() { }
 
+  /// Called when the platform's desired brightness changes.
+  ///
+  /// For example, starting in Android Pie, battery saver mode asks all apps to
+  /// render in a "dark mode". That change is reflected in this property.
+  ///
+  /// Not all platforms necessarily support a concept of brightness mode. Those
+  /// platforms will report [Brightness.light] in this property.
+  /// 
+  /// See also:
+  /// 
+  ///  * [MediaQuery.of], which provides a similar service with less
+  ///    boilerplate.
```

Comment:
Instead of "less boilerplate" you could mention that MediaQuery is an inherited widget whose dependents are automatically rebuilt..

Reply:
I changed it to your recommendation.